src/api/models/classification.py

```diff
@@ -0,0 +1,155 @@
+
+
+class Classifier(BaseModel):
+    name = models.CharField(max_length=50)
+    version = models.CharField(max_length=11)
+    patch_size = models.IntegerField(help_text="Number of pixels")
+    num_points = models.IntegerField(default=25)
+    description = models.TextField(max_length=1000, blank=True)
+    labels = models.ManyToManyField(Label, related_name="classifiers")
+
+    class Meta:
+        db_table = "class_classifier"
+
+
+class Image(BaseModel):
+    collect_record_id = models.UUIDField()
+    observation = models.ForeignKey(
+        ObsBenthicPhotoQuadrat, null=True, blank=True, on_delete=models.SET_NULL
+    )
+
+    image = models.ImageField(upload_to="", storage=select_image_storage)
+    original_image_checksum = models.CharField(max_length=64, blank=True, null=True)
+    thumbnail = models.ImageField(upload_to="", storage=select_image_storage, null=True, blank=True)
+
+    name = models.CharField("Name", max_length=200, blank=True, null=True)
+    photo_timestamp = models.DateTimeField(null=True, blank=True)
+    location = models.PointField(srid=4326, blank=True, null=True)
+    comments = models.TextField(max_length=1000, blank=True, null=True)
+    data = models.JSONField("Data", null=True, blank=True)
+
+    class Meta:
+        db_table = "class_image"
+
+    def __str__(self):
+        return f"{self.name} - {self.photo_timestamp}"
+
+    @property
+    def collect_record(self):
+        return CollectRecord.objects.get_or_none(id=self.collect_record_id)
+
+    @property
+    def project(self):
+        cr = self.collect_record
+        if cr:
+            return cr.project
+
+        obs = self.observation
+        return obs.benthic_photo_quadrat_transect.quadrat_transect.sample_event.site.project
+
+
+class Point(BaseModel):
+    row = models.IntegerField()
+    column = models.IntegerField()
+    point_number = models.IntegerField()
+    image = models.ForeignKey(Image, on_delete=models.CASCADE, related_name="points")
+
+    class Meta:
+        db_table = "class_point"
+
+
+class Annotation(BaseModel):
+    point = models.ForeignKey(Point, on_delete=models.CASCADE, editable=False)
+    label = models.ForeignKey(Label, related_name="annotations", on_delete=models.CASCADE)
+    classifier = models.ForeignKey(
+        Classifier, null=True, on_delete=models.CASCADE, related_name="annotations"
+    )
+    score = models.PositiveSmallIntegerField(default=0)
+    is_confirmed = models.BooleanField(default=False)
+
+    class Meta:
+        db_table = "class_annotation"
+        indexes = [
+            models.Index(
+                fields=["point", "label"],
+                name="unq_conf_anno_idx",
+                condition=models.Q(is_confirmed=True),
+            )
+        ]
+
+    def __str__(self):
+        return f"{self.label} - {self.score} - {self.is_confirmed}"
+
+    @property
+    def is_human_created(self):
+        return self.classifier is None and self.is_confirmed
```

Comment:
Now I'm wondering about a possible future edge case in which we would want to have an annotation that is human-created (`self.classifier is None`) but not confirmed (perhaps because they're not sure).